### PR TITLE
feat(caip): add caip data for binance, kava, terra, secret

### PR DESCRIPTION
**Pull request by pastaghost**
This adds CAIP data for Cosmos network assets {binance, kava, terra, secret}.

### CODE REVIEW

File: packages/caip/src/constants.ts [hunk 1]
```diff
@@ -14,6 +14,10 @@ export const foxyAssetId: AssetId = 'eip155:1/erc20:0xdc49108ce5c57bc3408c3a5e95
 export const cosmosAssetId: AssetId = 'cosmos:cosmoshub-4/slip44:118'
 export const osmosisAssetId: AssetId = 'cosmos:osmosis-1/slip44:118'
 export const thorchainAssetId: AssetId = 'cosmos:thorchain-mainnet-v1/slip44:931'
+export const binanceAssetId: AssetId = 'cosmos:binance-chain-tigris/slip44:714'
+export const kavaAsssetId: AssetId = 'cosmos:kava_2222-10/slip44:459'
+export const terraAsssetId: AssetId = 'cosmos:phoenix-1/slip44:330'
+export const secretAsssetId: AssetId = 'cosmos:secret-4/slip44:529'
```
Comment (0xApotheosis) [inline packages/caip/src/constants.ts:20]: Too many s's in "asset".

**Review by 0xApotheosis — CHANGES_REQUESTED** [review]

File: packages/caip/src/constants.ts (hunk 1 above)
Comment (pastaghost) [reply packages/caip/src/constants.ts:20]: I was using snake case ;)

Comment (0xApotheosis) [reply packages/caip/src/constants.ts]: 😁

**Review by 0xApotheosis — APPROVED** [review]
Now it's a little less slippery 🐍

**Pull request merged**